packages/manager/apps/dedicated/client/app/dedicated/server/details/server.constants.js

```diff
@@ -29,10 +29,17 @@ export const OPERATING_SYSTEM_ENUM = {
   BRING_YOUR_OWN_IMAGE: 'byoi_32',
 };
 
+export const CPANEL_ALERT_LINK = {
+  url:
+    'https://docs.cpanel.net/knowledge-base/cpanel-product/cpanel-deprecation-plan/',
+  title: "cPanel's deprecation plan",
```

Comment:
**question:** Dont we need translation for this?

Reply:
No it's the same link and text for all languages